.goreleaser.yml

```diff
@@ -194,3 +194,6 @@ brews:
       output = shell_output("#{bin}/epinio settings update-ca 2>&1")
       assert_match "failed to get kube config", output
       assert_match "no configuration has been provided", output
+release:
+  extra_files:
+    - glob: ./helm-charts/chart/epinio/crds/*
```

Comment:
I'm not sure if the git submodule is available here

Reply:
I guess this should do the trick: https://github.com/epinio/epinio/blob/151121bfb2aed5fb89a8a851d1c38f187a10e7ee/.github/workflows/release.yml#L18

Reply:
I'll approve (LGTM) but you can try it locally if you need to, maybe just checking out a different submodule version.

The `goreleaser --snapshot --rm-dist` command should do the trick.